ui/src/album/AlbumDetails.js

```diff
@@ -112,15 +117,32 @@ const AlbumComment = ({ record }) => {
   const classes = useStyles()
   const [expanded, setExpanded] = React.useState(false)
 
+  const reqClassName = classes.externalLinks
+  console.log(reqClassName)
+  const linkify = useCallback(
+    (text) => {
+      // eslint-disable-next-line
+      const urlRegex =
+        // eslint-disable-next-line
+        /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gi
+      return text.replace(urlRegex, function (url) {
+        return (
+          '<a class=' + reqClassName + ' href="' + url + '">' + url + '</a>'
```

Comment:
I think the vanilla anchor tag works, but to ensure continuity in how links appear and behave (e.g. :visited), I think a `<Link>` component without extra styles in this file. I would like to test this out, but at the moment I am not finished with setting up the linting stuff in my dev-environment.

More about accessibility in links: https://www.smashingmagazine.com/2021/12/designing-better-links-websites-emails-guideline/#link-accessibility